k8s/pkg/apis/longhorn/v1beta2/orphan.go

```diff
@@ -5,12 +5,15 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 type OrphanType string
 
 const (
-	OrphanTypeReplica = OrphanType("replica")
+	OrphanTypeReplicaDataStore = OrphanType("replica")
```

Comment:
If we are unable to find a better name, we can continue using `OrphanTypeReplicaData`.
`DataStore` is already used for the dataStore in the control plane. The naming is quite confusing. 
```suggestion
	OrphanTypeReplicaDataStore = OrphanType("replica")
```